.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -34,9 +34,9 @@ Short description explaining the high-level reason for the pull request
 * [ ] Code can be automatically merged (no conflicts)
 * [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
 * [ ] Passes all existing automated tests
-* [ ] New functions include new tests
+* [ ] Any *change* in functionality is tested
 * [ ] New functions are documented (with a description, list of inputs, and expected output)
 * [ ] Placeholder code is flagged
 * [ ] Visually tested in supported browsers and devices
 * [ ] Project documentation has been updated
-* [ ] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:
+* [ ] Reviewers requested with the [Reviewer tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:
```

Comment:
How about changing this link to [this](https://help.github.com/articles/about-pull-request-reviews/) help article instead?